ntbs-service/Pages/ContactDetails/Index.cshtml.cs

```diff
@@ -49,9 +51,29 @@ public async Task<IActionResult> OnGetAsync()
 
             ViewData["Referer"] = StringValues.IsNullOrEmpty(Request.Headers["Referer"])
                 ? "/"
-                : Request.Headers["Referer"].ToString(); 
+                : Request.Headers["Referer"].ToString();
 
+            PrepareBreadcrumbs();
+            
             return Page();
         }
+
+        private void PrepareBreadcrumbs()
+        {
+            var regionCode = ContactDetails.CaseManagerTbServices.FirstOrDefault()?.TbService.PHECCode;
+            var breadcrumbs = new List<Breadcrumb>
+            {
+                new Breadcrumb {Label = "Service Directory", Url = "/ServiceDirectory"}
+            };
+
+
+            if (regionCode != null)
+            {
+                breadcrumbs.Add(new Breadcrumb {Label = "Region", Url = $"/ServiceDirectory/Region/{regionCode}"});
```

Comment:
I think the label wants to be the region name, rather than the generic term.
Ditto for "Contact details" - let's put the name of the person there